topo/node/cptx/cptx_test.go

```diff
@@ -44,6 +48,33 @@ func (f *fakeWatch) ResultChan() <-chan watch.Event {
 	return eCh
 }
 
+// removeCommentsFromConfig removes comment lines from a JunOS config file
+// and returns the remaining config in an io.Reader.
+// Using scrapl_cfg_testing results in an EOF error when config includes comments.
+// Comments in config files are not problematic when using kne_cli (not testing).
+// This is a simple implementation that only removes lines that are entirely comments.
+func removeCommentsFromConfig(f *os.File) (io.Reader, error) {
```

Comment:
this func is a good candidate to be a test helper: go/go-style/decisions#mark-test-helpers

then the signature can be:

func removeCommentsFromConfig(t *testing.T, io.Reader) io.Reader {}